tests.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
+
+SITE_LOCATION = "http://localhost:5000"
+
+
+def url_ok(url):
```

Comment:
test functions names in pytest must start with `test_` and describe what is being tested.

The functions should typically contain some kind of an assertion

Reply:
the issue was addressed and fixed as required and explained in the comments above.